src/main/java/anna_gontari/Inheritance/testingInheritance.java

```diff
@@ -0,0 +1,39 @@
+package anna_gontari.Inheritance;
+
+import java.util.ArrayList;
+
+public class testingInheritance {
+    public static void main(String[] args) {
+        Dog myDog = new Dog("dachshund", 3 );
```

Comment:
separate all main implementation on different method, method main should be easy to read